addons/spreadsheet_dashboard/tests/test_spreadsheet_dashboard.py

```diff
@@ -17,3 +18,9 @@ def test_create_with_default_values(self):
             dashboard.raw,
             b'{"version": 1, "sheets": [{"id": "sheet1", "name": "Sheet1"}]}',
         )
+
+    def test_unlink_prevent_spreadsheet_group(self):
+        website_group = self.env.ref('spreadsheet_dashboard.spreadsheet_dashboard_group_website', False)
```

Comment:
I would have created a new group and its ir.model.data from scratch.
Tests depending on data (or, worse, demo data) are more likely to just be deleted if the data is removed some day in the future, instead of being adapted.

Here with `False`, the test won't even fail and will not test anything anymore if the data is removed